robot_target_client: # Node name
  ros__parameters:
    use_sim_time: true
    waypoint1: 'green'
    waypoint2: 'red'
    waypoint3: 'orange'
    waypoint4: 'purple'
    waypoint5: 'blue'